# 能量机关

# 击打点是否占领
bool isAttackPointOccupied
# 小能量机关是否已经激活
bool isSmallActive
# 大能量机关是否已经激活
bool isBigActive